pnpm-lock.yaml

```diff
@@ -18818,26 +20362,49 @@ packages:
       use-sidecar: 1.1.2_sudpmbbyhqtxq6t4xf6jlicdem
     dev: false
 
-  /react-router-dom/6.5.0_sfoxds7t5ydpegc3knd667wn6m:
-    resolution: {integrity: sha512-/XzRc5fq80gW1ctiIGilyKFZC/j4kfe75uivMsTChFbkvrK4ZrF3P3cGIc1f/SSkQ4JiJozPrf+AwUHHWVehVg==}
+  /react-router-dom/6.7.0_sfoxds7t5ydpegc3knd667wn6m:
```

Comment:
Here it is. Remix uses react-router-dom 6.7.0. You can fix it without caret until next upgrade.